feature-libs/product-configurator/rulebased/core/state/reducers/configurator.reducer.spec.ts

```diff
@@ -352,22 +352,20 @@ describe('Configurator reducer', () => {
     });
   });
 
-  describe('GetConfigurationOverviewSuccess action', () => {
-    it('should put configuration overview into the state', () => {
-      const priceSummary: Configurator.PriceSummary = {};
-      const overview: Configurator.Overview = {
-        configId: CONFIG_ID,
-        productCode: PRODUCT_CODE,
-        priceSummary: priceSummary,
-      };
-      const action = new ConfiguratorActions.GetConfigurationOverviewSuccess({
+  describe('SearchVariantsSuccess action', () => {
+    it('should put the variants list into the state', () => {
+      const variants: Configurator.Variant[] = [
+        {
+          productCode: PRODUCT_CODE,
+        },
+      ];
+      const action = new ConfiguratorActions.SearchVariantsSuccess({
         ownerKey: configuration.owner.key,
-        overview: overview,
```

Comment:
did you intend to replace this test?

Reply:
Yes, it's a duplicate. Its content is available split into 2 tests a couple of lines below.
I guess someone refactored that and forgot to delete before